src/components/common/YourCompany/YourCompany.vue

```diff
@@ -258,13 +242,14 @@
 
     <!-- Recognition Date and Time (alterations and BEN corrections only) -->
     <div class="section-container" v-if="isAlterationFiling || isBenIaCorrectionFiling">
+      <v-divider class="mx-4 my-1" />
       <v-row no-gutters>
         <v-col cols="3" class="pr-2">
           <label><strong>Recognition Date and Time</strong></label>
         </v-col>
 
         <v-col cols="9">
-          <div class="info-text">{{ recognitionDateTime }}</div>
+          <div class="info-text">{{ recognitionDate }}</div>
```

Comment:
Well, now the label on line 248 is incorrect. Is it intentional to only show the date, and the not the date-time, for alteration filings?

Reply:
Yes kind of... the shared date picker only returns a date no time so if you are showing a corrected start date there won't be a time, and the UXPin I looked at only show a date no time in the designs

Reply:
This line of code appears to display the date (no time) for some filing types. Previously this was the date-time. I'm not aware that this should be changed.

Also, the label on line 248 says "date and time" but only date is being displayed.

Do you have a requirement to change the alteration filing and BEN IA correction filing?

I'm thinking you need to revert the changes to the recognitionDate getter below.

Reply:
Okay I will revert the changes to the recognitionDate getter

Reply:
This component is used by alteration filings as well as firm correction filings, right? Then it needs to handle both.

Reply:
Code looks good now 👍